source/services/odnoklassniki.js

```diff
@@ -20,7 +20,7 @@ const odnoklassniki = {
     popupHeight: 400,
 };
 
-set(window, 'ODKL.updateCount', (index, counter) => {
+set(global, 'ODKL.updateCount', (index, counter) => {
```

Comment:
> But why window was used as storage?

That's a good question.

Let's find the placed where it's used:

- https://github.com/ilyabirman/Likely/blob/7e5daac7a15179a58c0ca7397bc73d3e9d518be1/source/service.js#L21-L25
- https://github.com/ilyabirman/Likely/blob/7e5daac7a15179a58c0ca7397bc73d3e9d518be1/source/services/vk.js#L23

@iamakulov